### PR TITLE
Add Fabric, Aurora, Corda to DAML-on-X Integration List

### PR DESCRIPTION
**Pull request by dasormeter**
Superceded by https://github.com/digital-asset/daml/pull/1813

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/docs/source/support/release-notes.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by bethaitman** [comment]
I think you might want to preview this using `/docs/scripts/live-preview.sh` - looks like there are some formatting issues!

File: docs/source/daml-integration-kit/index.rst [hunk 1]
```diff
@@ -101,18 +101,34 @@ support for running DAML.
 
    * - Ledger
      - Status
-     - Developer
+     - Integration Support
```
Comment (bethaitman) [inline docs/source/daml-integration-kit/index.rst]: I don't quite understand why this changed?

File: docs/source/daml-integration-kit/index.rst [hunk 2]
```diff
@@ -101,18 +101,34 @@ support for running DAML.
...
      - More
+
```
Comment (bethaitman) [inline docs/source/daml-integration-kit/index.rst]: This line will probably break the list

File: docs/source/daml-integration-kit/index.rst [hunk 3]
```diff
@@ -101,18 +101,34 @@ support for running DAML.
...
        <http://hub.digitalasset.com/press-release/digital-asset-daml-smart-contract-language-now-extended-to-vmware-blockchain>`__
    * - `Hyperledger Sawtooth <https://sawtooth.hyperledger.org/>`__
      - In Development
      - `Blockchain Technology Partners <https://blockchaintp.com/>`__
-     - `press release (April 16th 2019)
+     - `press release (April 2019)
        <https://www.hyperledger.org/blog/2019/04/16/daml-smart-contracts-coming-to-hyperledger-sawtooth>`__
+   * - `Hyperledger Fabric <https://www.hyperledger.org/projects/fabric>`__
+     - In Development
+     - ``Hacera Unbounded Network <https://unbounded.network/>`__
```
Comment (bethaitman) [inline docs/source/daml-integration-kit/index.rst]:
```suggestion
     - `Hacera Unbounded Network <https://unbounded.network/>`__
```

File: docs/source/daml-integration-kit/index.rst [hunk 4]
```diff
@@ -101,18 +101,34 @@ support for running DAML.
...
+     - `press release (June 2019)
+       <https://hub.digitalasset.com/press-release/digital-asset-announces-daml-partner-integrations-with-hyperledger-fabric-r3-corda-and-amazon-aurora>`__
+   * - `R3 Corda <https://www.corda.net>`__
+     - In Development
+     - `Blockchain Technology Partners <https://blockchaintp.com/>`__
```
Comment (gleber-da) [inline docs/source/daml-integration-kit/index.rst]:
```suggestion
     - `Digital Asset <https://digitalasset.com/>`__
```

File: docs/source/daml-integration-kit/index.rst (hunk 1 above)
Comment (dasormeter) [reply docs/source/daml-integration-kit/index.rst]: @bethaitman -- from the PR it is mentioned the Integration company that would run the network upon which a solution would be hosted, which is slightly different to Developer.  Not sure if there are actually two things to show here -- who performed the initial development and who will support integration in a running environment.  Probably needs a Chris/Simon/Shaul conversation - I'll also bring up in standup today.

Comment (bethaitman) [reply docs/source/daml-integration-kit/index.rst]: Cool, thanks for the clarification. I don't mind which you go for (or if you want to list both the developer and who would host the integration) just as long as the information is clear. I wonder if something like "Integration hosted by" or "Ledger hosted by" might be a better name for the latter.

**Comment by S11001001** [comment]
@remyhaemmerle-da It's not that, it's that Github seems to be choosing the wrong merge-base to use for the diff and commit logs. `git log origin/master..origin/HOTFIX-ledger-integration-update-roadmap` shows a correct merge-base having been chosen, and `git diff origin/master...origin/HOTFIX-ledger-integration-update-roadmap` shows the diff that GH should be showing.

```sh
$ git log -1 `git merge-base origin/master origin/HOTFIX-ledger-integration-update-roadmap`
commit 8f006e271472f14ee607ac7725990e8c1a24f11d
Author: Remy
Date:   Wed Jun 19 18:21:59 2019 +0200

    minor fix for scala typecheck  (#1764)
```

@dasormeter might need to open a new PR to fix.

**Comment by bethaitman** [comment]
@brianhansen-da - I'd love to get this finished, as we've just got a question on the public channel about it. Were you going to open a new PR to fix it?

**Comment by meiersi-da** [comment]
@bethaitman : did you mean @dasormeter ?

**Comment by bethaitman** [comment]
Ah, yes, oops, not sure how I managed that.

**Comment by dasormeter** [comment]
closing -- superceded by https://github.com/digital-asset/daml/pull/1813

**Pull request closed without merging**